workos/directory_sync.py

```diff
@@ -111,12 +111,13 @@ def get_group(self, group):
         )
 
     def list_directories(
-        self, domain=None, search=None, limit=RESPONSE_LIMIT, before=None, after=None
+        self, domain=None, organization=None, search=None, limit=RESPONSE_LIMIT, before=None, after=None
```

Comment:
I think `organization` should technically be added to the end, as if a caller is calling `list_directories` with positional arguments (instead of named arguments) then shifting the existing arguments would be a breaking change.

I think Python has ways to force callers to use named arguments, so that might be something we want to look into at some point.

Reply:
For sure, and I just switched organization to be the last param